packages/drawer/src/index.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <transition
+    name="el-drawer-fade"
+    @after-enter="afterEnter"
+    @after-leave="afterLeave"
+  >
+    <div
+      v-show="visible"
+      ref="root"
+      class="el-drawer__wrapper"
+      tabindex="-1"
+    >
+      <div
+        class="el-drawer__container"
+        :class="visible && 'el-drawer__open'"
+        tabindex="-1"
+        role="document"
+        @click.self="handleWrapperClick"
+      >
+        <div
+          ref="drawer"
+          aria-modal="true"
+          aria-labelledby="el-drawer__title"
+          :aria-label="title"
+          class="el-drawer"
+          :class="[direction, customClass]"
+          :style="isHorizontal ? `width: ${size}` : `height: ${size}`"
+          role="dialog"
+          tabindex="-1"
+        >
+          <header
+            v-if="withHeader"
+            id="el-drawer__title"
+            class="el-drawer__header"
+          >
+            <slot name="title">
+              <span role="heading" tabindex="0" :title="title">{{
+                title
+              }}</span>
+            </slot>
+            <button
+              v-if="showClose"
+              :aria-label="`close ${title || 'drawer'}`"
+              class="el-drawer__close-btn"
+              type="button"
+              @click="closeDrawer"
+            >
+              <i class="el-dialog__close el-icon el-icon-close"></i>
```

Comment:
`el-dialog__close` maybe should be `el-draw__close`, I think it's legacy.

Reply:
OK！I will fix it。